eth/api.go

```diff
@@ -1744,6 +1745,38 @@ func (api *PublicDebugAPI) Metrics(raw bool) (map[string]interface{}, error) {
 	return out, nil
 }
 
+// Verbosity implements api method debug_verbosity, enabling setting
+// global logging verbosity on the fly.
+// Note that it will NOT allow setting verbosity '0', which is effectively 'off'.
+// In place of ability to receive 0 as a functional parameter, debug.verbosity() -> debug.verbosity(0) -> glog.GetVerbosity().
+// creates a shorthand/convenience method as a "getter" function returning the current value
+// of the verbosity setting.
+func (api *PublicDebugAPI) Verbosity(n uint64) (int, error) {
+	nint := int(n)
+	if nint == 0 {
+		s := glog.GetVerbosity().String()
```

Comment:
Instead of unnecessary conversion to string and back to int, simple type conversion is possible: `int(*glog.GetVerbosity())`. I don't think that underlying type will change often, so the "fragility" of the conversion shouldn't be a problem.